src/NBitcoin/Consensus.cs

```diff
@@ -30,6 +30,26 @@ public class ConsensusOptions
         {
         }
 
+        /// <summary>
+        /// How many blocks should be on top of the block with coinbase transaction until it's outputs are considered spendable.
+        /// </summary>
+        public long CoinbaseMaturity { get; set; }
+
+        /// <summary>
+        /// Amount of coins mined when a new network is bootstrapped.
```

Comment:
maybe add to the comment that zero means disabled? (same for the other guy)

Reply:
sure